src/main/java/org/commcare/formplayer/beans/AuthenticatedRequestBean.java

```diff
@@ -22,6 +22,7 @@ public class AuthenticatedRequestBean {
     private int timezoneOffsetMillis = -1;
     private String timezoneFromBrowser = null;
     private String windowWidth;
+    private String locale;
```

Comment:
This doesn't seem like it's getting used anymore, think we can remove all bean model changes with the updated implementation.

Reply:
this is solely so that the test can pass!

Reply:
nevermind! this was removed